### PR TITLE
Fixing some small issues in docs

### PR DESCRIPTION
**Pull request by apetkau**
Fixes to some small issues in documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,15 +66,15 @@ IRIDA Next pipelines attempt to follow as much as possible the [nf-core][] guide
...
+While we do follow nf-core as closely as possible, there are a number of differences or additions we make for integrating pipelines into IRIDA Next. Hence, some of the nf-core requirements may not apply if you do not intend to contribute the pipeline to nf-core.
 
 ## 1.1. Developing a pipeline to contribute to nf-core
 
 If you intend to develop a pipeline to contribute to nf-core, please follow all the [nf-core requirements][nf-core-requirements] in addition to the requirements listed in this guide.
 
 ## 1.2. Developing a pipeline which won't be contributed to nf-core
 
-If, instead, the pipeline won't be contributed to nf-core, then the only following set of [nf-core requirements][nf-core-requirements] would need to be followed in addition to the [if the guidelines don't fit][nf-core-external-development] section of the nf-core guidelines, [using nf-core components outside of nf-core][nf-core-outside-nf-core] section, and the below IRIDA Next requirements sections.
+If, instead, the pipeline won't be contributed to nf-core, then the only following set of nf-core requirements would need to be followed along with the below IRIDA Next requirements sections.
```
Comment (kylacochrane) [inline README.md:77]: If, instead, the pipeline won't be contributed to nf-core, then only the following set of nf-core requirements would need to be followed, along with the IRIDA Next requirements sections below.

**Review by kylacochrane — CHANGES_REQUESTED** [review]
Only one small suggestion for the updated sentence at line 77 :)

File: README.md (hunk 1 above)
Comment (apetkau) [reply README.md]: Thanks so much. Fixed in ee1165dcaf58a14ac12ec5cac35ef07f5ed7c374

**Review by kylacochrane — APPROVED** [review]
Looks great Aaron :)

**Pull request merged**